tsl/test/sql/.gitignore

```diff
@@ -5,3 +5,8 @@
 /ordered_append-*.sql
 /plan_skip_scan-*.sql
 /transparent_decompression*-1[3-9].sql
+/chunk_utils_internal-*.sql
```

Comment:
Can we follow the approach by https://github.com/timescale/timescaledb/pull/8386 and https://github.com/timescale/timescaledb/pull/8379 to avoid the test changes related to not null constraints?